docs/core/docker/snippets/8.0/App/Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM mcr.microsoft.com/dotnet/sdk:8.0@sha256:35792ea4ad1db051981f62b313f1be3b46b1f45cadbaa3c288cd0d3056eefb83 AS build-env
+FROM mcr.microsoft.com/dotnet/sdk:8.0@sha256:35792ea4ad1db051981f62b313f1be3b46b1f45cadbaa3c288cd0d3056eefb83 AS build
 WORKDIR /App
 
 # Copy everything
 COPY . ./
 # Restore as distinct layers
 RUN dotnet restore
 # Build and publish a release
-RUN dotnet publish -c Release -o out
+RUN dotnet publish -c Release --property:PublishDir=out
```

Comment:
Why the switch from `-o` here? Generally speaking if there is a first-class CLI arg for a property we should use it instead.

Reply:
also nit: `-c Release` is the default for `dotnet publish`, it doesn't need to be specified.

Reply:
There was a specific issue related to this, see: https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/7.0/solution-level-output-no-longer-valid. Apparently, we shouldn't be using output anymore?